clients/lsp-php.el

```diff
@@ -378,20 +378,15 @@ already present."
   :link '(url-link "https://github.com/phpactor/phpactor")
   :group 'lsp-mode)
 
-(defcustom lsp-phpactor-path nil
+(defcustom lsp-phpactor-path (or (executable-find "phpactor")
```

Comment:
execute-find should not be executed on the top level. This means that it will be executed for all lsp-mode users. All calculations should happen in the lsp-stdio-connection lambda

Reply:
Ah that makes sense, apologies! Amending